homeassistant/components/microbees/entity.py

```diff
@@ -0,0 +1,30 @@
+"""Base entity for microBees."""
+
+from microBeesPy.microbees import Actuator, Bee
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import MicroBeesUpdateCoordinator
+
+
+class MicroBeesEntity(CoordinatorEntity[MicroBeesUpdateCoordinator]):
+    """Base class for microBees entities."""
+
+    def __init__(
+        self, coordinator: MicroBeesUpdateCoordinator, act: Actuator, bee: Bee
+    ) -> None:
+        """Initialize the microBees entity."""
+        super().__init__(coordinator)
+        self._attr_available = False
```

Comment:
Instead of declaring it like this, wouldn't it make more sense to have an `available` property that checks if the `bee_id` is present in `self.coordiantor.data`? So if the coordinator doesn't return the bee_id, it would go unavailable

Reply:
> Can be set outside of the coordinator

Wha do you mean?

Reply:
I meant constructor, whoops